index.bs

```diff
@@ -2581,6 +2581,38 @@ value explicitly. <em>block-size</em> is defined to be 128
 sample-frames which corresponds to roughly 3ms at a sample-rate of
 44.1 kHz.
 
+<h4 id="AudioNode-tail">AudioNode tail-time</h4>
+
+An {{AudioNode}} can have a <dfn>tail-time</dfn> that is greater than zero. This
```

Comment:
You're defining tail-time here, but that's doesn't actually say what it is.  Need t move the actual definition of tail-time to here.  I guess we don't really have a definition of just tail-time.  It's tail-time reference.

In any case, we need a better definition.

Reply:
Sorry, you actually define it in the next sentence. Perhaps combine the two sentences into one longer sentence.  Or maybe something like:

Each AudioNode has a tail-time.  The `<dfn>tail-time</dfn> is the time for silent input to be produced at the output of the node.  Some nodes like the BiquadFilterNode have mathematically infinite tail times; it is at the discretion of the UA to determine an appropriate finite value for the tail time.

Reply:
BiquadFilterNode and others already describe their tail-times reasonably well.
One thing that is perhaps not perfectly clear from their descriptions is that the tail-time applies only after input changes from non-silent to silent.

The mention of "internal processing state" is dropped with the lifetime section.  It could be retained if that helps with the concept of "affects":

"Nodes have a non-zero tail-time if they have internal processing state such that input received at one time affects the output at later times.  Such nodes that have received non-silent input continue to produce non-silent output for the length of the tail-time after the input becomes silent."

Reply:
I've picked @karlt sentence, that is more generic.